wolfssl/wolfcrypt/settings.h

```diff
@@ -2026,6 +2041,16 @@ extern void uITRON4_free(void *p) ;
     #endif
 #endif
 
+#if defined(WOLFSSL_HARDEN_TLS) && ECC_MIN_KEY_SZ < 224 && \
```

Comment:
What about when WOLFSSL_HARDEN_TLS is 128?

Reply:
If a user wants to specify a `ECC_MIN_KEY_SZ` when using `WOLFSSL_HARDEN_TLS=128` then we can allow that (the 128 bits is only a SHOULD in the RFC). The 112 bits requirement is a MUST which is why I mandated only the 112 bit requirement.

Reply:
Thanks!